### PR TITLE
CA, SC

### PR DESCRIPTION
**Pull request by gilmourj**

### CODE REVIEW

File: urls.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ filter: ocr,clean-new-lines
 kind: url
 name: California
 url: https://update.covid19.ca.gov/
-filter: css:div.container:contains("California now has"),html2text,strip
+filter: css:div.container:contains("California has"),html2text,strip
```
Comment (zachlipton) [inline urls.yaml]:
```suggestion
filter: css:div.container:contains("The number of COVID-related deaths"),html2text,strip
```

**Review by zachlipton — COMMENTED** [review]
Thank you! Rebma told me you were going to help update these, and then next thing I knew your PR was here

This looks great! Only tweak I'm making is tightening up the filter for California to cover just a smaller part of the page. It's ok if the filter doesn't cover every single number, and slurping up a lot can generate more false positives, but it's a balancing act.

Please @ me on Slack in #coding-urlwatch if you have any questions about this or have development environment issues. Thanks so much

**Comment by space-buzzer** [comment]
My addition here is that you should write the expected output in the PR, e.g., 
CA was broken, now it's:
```
output
```

**Comment by gilmourj** [comment]
Thanks for the tips! I'll be more specific with filters and PR language going forward. Appreciate the help.

**Pull request merged**